### PR TITLE
hasp map usage corrected

### PR DESCRIPTION
**Pull request by nextLane**
Trying to understand how Cassovary works, found this compile time error while trying examples. Fixed it.

### CODE REVIEW

File: cassovary-examples/src/main/java/RandomWalkJava.java [hunk 1]
```diff
@@ -91,7 +91,7 @@ public static void main(String[] args) {
   @SuppressWarnings("unchecked")
   private static Map<Integer, Integer> scalaIntsMapToJavaMap(scala.collection.Map<Object, Object> map) {
     Map<Object, Object> javaMap = JavaConverters$.MODULE$.mapAsJavaMapConverter(map).asJava();
-    HashMap<Integer, Integer> javaIntsMap = new HashMap<>();
+    HashMap<Integer, Integer> javaIntsMap = new HashMap<Integer, Integer>();
```
Comment (bmckown) [inline cassovary-examples/src/main/java/RandomWalkJava.java]: Which version of java are you using?  To my knowledge that should compile for java 7 and up.  

http://docs.oracle.com/javase/7/docs/technotes/guides/language/type-inference-generic-instance-creation.html

Comment (AnishShah) [inline cassovary-examples/src/main/java/RandomWalkJava.java]: Yep. HashMap<>() compiles for Java 7 and up.
And after pull #124 , cassovary works for Java 7 and up.

cc @pankajgupta

Comment (szymonm) [inline cassovary-examples/src/main/java/RandomWalkJava.java]: yes, diamond notation was introduced in 7

Comment (nextLane) [inline cassovary-examples/src/main/java/RandomWalkJava.java]: Yes, my bad, I was using JAVA 6, updated to 8. Works well now.

**Comment by pankajgupta** [comment]
Updated README to clarify this.

**Pull request closed without merging**